event_sale_session/static/src/js/EventConfiguratorFormController.esm.js

```diff
@@ -0,0 +1,36 @@
+/** @odoo-module **/
+/*
+    Copyright 2021 Moka Tourisme (https://www.mokatourisme.fr)
+    Copyright 2023 Tecnativa - Carlos Roca
+    @author Iván Todorovich <ivan.todorovich@gmail.com>
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+*/
+import {RelationalModel} from "@web/views/relational_model";
+import {patch} from "@web/core/utils/patch";
+
+patch(RelationalModel.prototype, "event_sale_session.RelationalModel", {
+    setup(params, {action}) {
+        // Trick to keep model as the main EventConfiguratorRelationalModel is not exported
```

Comment:
Most probably Odoo would accept a PR making both `EventConfiguratorRelationalModel` and `EventConfiguratorRecord` exportables.